### DISCUSSION TITLE
Mass Product Attribute Update command for magento docker

### DISCUSSION THREAD

**Discussion by bhushanmeetanshi** [markshust/docker-magento#1192] [Q&A]
what is the command for Mass Product Update attributes for magento docker?

**Comment by bhushanmeetanshi** [comment]
@YevhenZvieriev Can you please guide me if u r available?

**Reply by YevhenZvieriev** [reply]
Hello, my friend.

Can you give me more details?

**Reply by bhushanmeetanshi** [reply]
I want to update 100 product qty with this command. is it feasible?


`bin/magento queue:consumers:start product_action_attribute.update`

**Reply by YevhenZvieriev** [reply]
Look at this one:

https://magento.stackexchange.com/questions/348529/product-attribute-update-que-stock

**Comment by bhushanmeetanshi** [comment]
Is any script need to create?

**Comment by bhushanmeetanshi** [comment]
Well I want to update Product qty for ex 100 product with mass product update from the magento admin panel then is any CLI command  available for magento docker so i can execute it?
